install.sh

```diff
@@ -110,6 +120,21 @@ prepare_user() {
   return 0
 }
 
+run_as_user() {
+  sudo -u $USER /bin/bash -c "
```

Comment:
Using `$@` is better. Will fix it. Thanks for advice.

But `-H env PATH=...` doesn't work because on CentOS 7, by default,

    Defaults    secure_path = /sbin:/bin:/usr/sbin:/usr/bin

is set in `/etc/sudoers`. If we want to make your command work, we need to edit `/etc/sudoers`. I think setting `ENV` in string is a little bit better than editing `/etc/sudoers`. How do you think?